lib/stagehand/staging/synchronizer.rb

```diff
@@ -32,10 +32,12 @@ def sync(limit = nil)
         deleted_count = 0
 
         Rails.logger.info "Syncing"
+
+        in_progress = CommitEntry.in_progress.pluck(:id)
```

Comment:
Isn't it theoretically possible that we can have a commit that starts after this query and before we start `iterate_autosyncable_entries`?

Reply:
Yeah, I was thinking I might need to initialize it in the loop if blank so that it is guaranteed to cover the entire time period that entries in the block modify.